app/views/groups/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<header class="header container primary-gradient-background">
+  <h1>Groups</h1>
+</header>
+<section>
+  <% if @groups.empty? %>
+    <div class="information padding border">
+      <h3 class="text-center">No groups yet.</h3>
+    </div>
+  <% else %>
+  <% @groups.each do |group| %>
+    <div class="d-flex information border gap">
+      <img src=<%= group.icon %> alt=<%= group.name %> class="icon">
+      <div class="container flex-fill justify-content-between">
+        <div class="d-flex justify-content-between">
+          <h3><%= link_to group.name, group_operations_path(group_id: group.id) %></h3>
+          <p>$<%= group_amount(group) %></p>
+        </div>
+        <small>
+          <%= group.created_at.strftime("%d %b %Y") %>
+        </small>
+      </div>
+    </div>
+  <% end %>
+  <% end %>
+  <div class="margin">
+    <%= link_to "Add group", new_group_path, class: "btn" %>
+  </div>
```

Comment:
- [OPTIONAL] As in the design, this button should be full width and fixed to the bottom of the screen. Please see the original design. The green element with the word scan is the button: 

Original Design |
:-------------------:|
![a1](https://user-images.githubusercontent.com/39523312/170297189-2779f2c1-dd8d-4423-a3c1-70987cbfa7ec.png)